services/web-app/data/teams.js

```diff
@@ -25,8 +25,12 @@ export const teams = {
     icon: IbmCloud,
     name: 'IBM Cloud'
   },
+  sterling: {
+    icon: Bee,
+    name: 'IBM Sterling'
+  },
   watson: {
     icon: Watson,
-    name: 'Watson'
+    name: 'IBM Watson'
```

Comment:
lol it's funny... @andreancardona and I debated on this as to whether people refer to it as "Watson" or "IBM Watson". I think what threw us off was for security, the icon is called `IbmSecurity` and for Watson it's just called `Watson`, so we followed suit with the names. Their website sometimes uses IBM Watson and other times uses things like Watson AI instead of IBM Watson AI. 😆  🤔

Reply:
Ya, it's officially IBM Watson. "Watson" by itself isn't trademarked.
![image](https://user-images.githubusercontent.com/1691245/148450807-7201c87e-4841-49b3-b0cc-69ee4f2d66a0.png)
![image](https://user-images.githubusercontent.com/1691245/148450816-4559edd9-7a83-46b8-bb33-33d13a9581b0.png)